apps/devportal/data/markdown/pages/learn/accelerate/xm-cloud/pre-development/security/head-application-security.md

```diff
@@ -80,6 +80,10 @@ const defaultConfig: JssConfig = {
 };
 ```
 
+### External Data Fetching
+
+When getting external data in a component using
```

Comment:
Looks like this may not be ready for a commit yet

Reply:
Removed, that shouldn't have made it to this PR